packages/realm-server/tests/realm-endpoints-test.ts

```diff
@@ -190,4109 +162,415 @@ module(basename(__filename), function () {
+      resetCatalogRealms();
+    });
 
-      setupPermissionedRealm(hooks, {
-        '*': ['read', 'write'],
-      });
+    hooks.beforeEach(async function () {
+      shimExternals(virtualNetwork);
+    });
 
-      let { getMessagesSince } = setupMatrixRoom(hooks);
+    setupPermissionedRealm(hooks, {
+      '*': ['read', 'write'],
+    });
 
-      async function startRealmServer(
-        dbAdapter: PgAdapter,
-        publisher: QueuePublisher,
-        runner: QueueRunner,
-      ) {
-        if (testRealm2) {
-          virtualNetwork.unmount(testRealm2.handle);
-        }
-        ({
-          seedRealm,
-          testRealm: testRealm2,
-          testRealmHttpServer: testRealmHttpServer2,
-        } = await runTestRealmServer({
-          virtualNetwork,
-          testRealmDir,
-          realmsRootPath: join(dir.name, 'realm_server_2'),
-          realmURL: testRealm2URL,
-          dbAdapter,
-          publisher,
-          runner,
-          matrixURL,
-        }));
+    let { getMessagesSince } = setupMatrixRoom(hooks);
 
-        await testRealm.logInToMatrix();
+    async function startRealmServer(
+      dbAdapter: PgAdapter,
+      publisher: QueuePublisher,
+      runner: QueueRunner,
+    ) {
+      if (testRealm2) {
+        virtualNetwork.unmount(testRealm2.handle);
       }
+      ({
+        seedRealm,
+        testRealm: testRealm2,
+        testRealmHttpServer: testRealmHttpServer2,
+      } = await runTestRealmServer({
+        virtualNetwork,
+        testRealmDir,
+        realmsRootPath: join(dir.name, 'realm_server_2'),
+        realmURL: testRealm2URL,
+        dbAdapter,
+        publisher,
+        runner,
+        matrixURL,
+      }));
+
+      await testRealm.logInToMatrix();
+    }
 
-      setupDB(hooks, {
-        beforeEach: async (_dbAdapter, _publisher, _runner) => {
-          dbAdapter2 = _dbAdapter;
-          publisher = _publisher;
-          runner = _runner;
-          testRealmDir = join(dir.name, 'realm_server_2', 'test');
-          ensureDirSync(testRealmDir);
-          copySync(join(__dirname, 'cards'), testRealmDir);
-          await startRealmServer(dbAdapter2, publisher, runner);
+    setupDB(hooks, {
+      beforeEach: async (_dbAdapter, _publisher, _runner) => {
+        dbAdapter2 = _dbAdapter;
+        publisher = _publisher;
+        runner = _runner;
+        testRealmDir = join(dir.name, 'realm_server_2', 'test');
+        ensureDirSync(testRealmDir);
+        copySync(join(__dirname, 'cards'), testRealmDir);
+        await startRealmServer(dbAdapter2, publisher, runner);
+      },
+      afterEach: async () => {
+        if (seedRealm) {
+          virtualNetwork.unmount(seedRealm.handle);
+        }
+        await closeServer(testRealmHttpServer2);
+      },
+    });
 
-          // To remove as part of CS-8014
-          testRealmEventSource = new eventSource(
-            `${testRealmHref}_message?testFileWatcher=watchexec`,
-          );
```

Comment:
Did you mean to delete this? 😳

Reply:
Yes. It was causing a test to fail and removing it made it succeed with no new failures.